flaskr/model.py

```diff
@@ -0,0 +1,17 @@
+from flask_sqlalchemy import SQLAlchemy
+from app import db
```

Comment:
If we've already imported these to `app.py`, and we are importing this file into that file, do we need to import them again here? Or can we take these out?

Reply:
you're right, we should be able to remove this